react-charts/donut-utilization-chart/step6.md

```diff
@@ -1,10 +1,10 @@
-Now we'll add a legend to the donut utilization chart...
+Change the donut utilization chart theme color.
 
-## Task
+1) <strong>Make sure the App.js file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component and add a peroperty to change the theme.</strong>

```

Comment:
Update this to follow what is in area-chart:

<img width="304" alt="Screen Shot 2019-12-05 at 10 44 18 PM" src="https://user-images.githubusercontent.com/20118816/70294380-3e4b6480-17b1-11ea-8288-d81ca8c3c685.png">

Reply:
think you meant to change 2) to follow the theme step of area-chart.  changing donut-util step to be:
2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>